initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1

```diff
@@ -222,9 +222,21 @@ Function boot_reqProc() {
 Function boot_reqKick() {
   echo "Booting up reqKick service..."
 
-  git clone https://github.com/Shippable/reqKick.git $REQKICK_DIR
+  $reqKick_tar_download_location="$env:TEMP/node.tar.gz"
+  Invoke-RestMethod "$REQKICK_DOWNLOAD_URL" `
+    -OutFile $reqKick_tar_download_location
+
+  gzip -df $reqKick_tar_download_location
+  $reqKick_tar=($reqKick_tar_download_location).replace(".gz", "").replace("\", "/")
+  pushd "$env:TEMP/"
+  tar -xf "$reqKick_tar" --force-local
+  mv $reqKick-$SHIPPABLE_RELEASE_VERSION/* $REQKICK_DIR
```

Comment:
I think this should be `mv reqKick-$SHIPPABLE_RELEASE_VERSION/* $REQKICK_DIR` (no `$` for reqKick, right?)